renderer/brave_wallet/brave_wallet_render_frame_observer.cc

```diff
@@ -41,6 +41,10 @@ void BraveWalletRenderFrameObserver::DidCreateScriptContext(
   native_javascript_handle_->AddJavaScriptObjectToFrame(context);
 }
 
+void BraveWalletRenderFrameObserver::DidFinishLoad() {
+  native_javascript_handle_->ConnectEvent();
```

Comment:
why is this being called in DidFinishLoad? This seems very late in the process and could be delayed significantly by slow loading page resources. Why wouldn't you call this immediately after creating the object in JS?

Reply:
also nit: I think this method name makes more sense as just `Connect`

Reply:
the whole firing of events was mimic like Metamask does. We don't need it to be called earlier.

Reply:
the code was fixed, but github didn't update so resolving